artcommon/artcommonlib/util.py

```diff
@@ -112,7 +112,8 @@ def get_assembly_release_date(assembly, group):
     assembly_release_date = None
     release_schedules = requests.get(f'{RELEASE_SCHEDULES}/{group}.z/?fields=all_ga_tasks', headers={'Accept': 'application/json'})
     for release in release_schedules.json()['all_ga_tasks']:
-        if assembly in release['name']:
+        # release['name'] usually has release name in value like "4.15.1 in Fast Channel", but for GA the value will be like "4.15 GA"
+        if (assembly in release['name']) or (assembly.split(".")[2] == '0' and "GA" in release['name']):  
```

Comment:
If `assembly` looks like `rc.1`, it will raise an error.

Reply:
updated in https://github.com/openshift-eng/art-tools/pull/423/commits/541cbab080d900f584957ff8ae6ec6b426c643bd